### PR TITLE
update grpc connector clients on message bus event

### PR DESCRIPTION
**Pull request by DFSko**
resolves happy-travel/agent-app-project#1501

### CODE REVIEW

**Review by AlexeyShok — APPROVED** [review]

File: Api/Services/Connectors/GrpcClientsStorageUpdater.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+
+    protected override Task ExecuteAsync(CancellationToken stoppingToken)
+    {
+        _connection.SubscribeAsync(MessageBusTopics.MarkupPolicyUpdated, (_, m) =>
+        {
+            _logger.LogSupplierUpdateMessageReceived();
+            
+            var supplier = JsonSerializer.Deserialize<SlimSupplier>(m.Message.Data);
+            if (supplier is null)
```
Comment (suhan3z) [inline Api/Services/Connectors/GrpcClientsStorageUpdater.cs:30]: Don't we need to log something, in this case?

File: Api/Services/Connectors/IGrpcClientsStorage.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+using HappyTravel.EdoContracts.Grpc.Services;
+using HappyTravel.SupplierOptionsClient.Models;
+
+namespace HappyTravel.Edo.Api.Services.Connectors;
+
+public interface IGrpcClientsStorage
+{
+    IConnectorGrpcService Get(SlimSupplier supplier);
```
Comment (suhan3z) [inline Api/Services/Connectors/IGrpcClientsStorage.cs:8]: Comments

Comment (DFSko) [reply Api/Services/Connectors/IGrpcClientsStorage.cs:8]: comment added

File: Api/Services/Connectors/GrpcClientsStorageUpdater.cs (hunk 1 above)
Comment (DFSko) [reply Api/Services/Connectors/GrpcClientsStorageUpdater.cs]: added logging

**Pull request merged**